slides/workshop_rap_for_managers_basics.qmd

```diff
@@ -257,15 +275,14 @@ Recyclable code is code that can be used in multiple projects. This means that y
 
 # What we expect from managers
 
-::: {.column width="40%"}
-- You should be familiar with our RAP standards, and we expect managers to ensure that their teams are following these standards
-- When your team reaches "baseline" (all elements of Good + Great), they will undergo an audit to ensure that the standards have been met
-- We then expect teams to begin working towards Best practice levels
+::: {.column width="45%"}
+- You should have sufficient levels of technical skills (R, SQL, Git) relevant to the work of your team to allow you to act as a technical leader and build capability in your team and the wider analyst community
+- You should be familiar with our RAP standards as a framework for analytical work, and we expect managers to ensure that their teams are following these standards
 - Each standard is described in detail on the Analysts' Guide - if this guidance is not suitable for your needs, <b>tell us</b>
 - <span style="color:#f67400;">**You should know how to advise your teams on each of these standards**</span>
```

Comment:
Add another point about knowing the support available (could be SSU poster, analysts guide page or RAP intranet page) and engaging intelligently with all the support that is on offer, our stuff and the central AF stuff.